test/routes/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import assert from 'power-assert';
+import request from 'supertest';
+import session from 'supertest-session';
+import app from 'app.js';
+import UserModel from 'models/User';
+
+describe('/routes/mail', ()=> {
+  let appSession;
```

Comment:
__[jshint]__
*  'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).